modules/ROOT/pages/update-streams-switching.adoc

```diff
@@ -0,0 +1,25 @@
+
+= Switching to a Different Stream
+
+In order to switch between the different streams of Fedora CoreOS
+(FCOS) a user can leverage the `rpm-ostree rebase` command.
+
+TIP: It may be a good idea to backup data under `/var` before switching streams.
+
+NOTE: Software updates generally follow the `next` -> `testing` -> `stable` flow, meaning `next` has the newest sofware and `stable` has the oldest sofware. Upstream software components are generally tested for upgrading, not downgrading, which means that upstream software typically can handle a data/configuration migration forward (upgrade), but not backwards (downgrade). For this reason it is typically safer to rebase from `stable` -> `testing` or `testing` -> `next`, but less safe to go the other direction.
+
+NOTE: Switching between streams may introduce regressions or bugs due to skipping update barriers. If you experience a regression please attempt a xref:manual-rollbacks.adoc[rollback].
+
+.Example switching FCOS to a different stream
+[source, bash]
+----
+# Stop the service that performs automatic updates
+sudo systemctl stop zincati.service
+
+# Perform the rebase to a different stream
+# Available streams: "stable", "testing", and "next".
```

Comment:
Nit: this has a trailing period, other comments don't (feel free to adopt the style you prefer).

Reply:
fixed